packages/SwingSet/src/lib/id.js

```diff
@@ -39,7 +37,9 @@ export function insistVatID(s) {
  * @returns {string} a vat ID string of the form "vNN" where NN is the index.
  */
 export function makeVatID(index) {
-  return `v${Nat(index)}`;
+  const vatID = `v${Nat(index)}`;
```

Comment:
`Nat` will throw rather than emit a non-Nat (`Nat(1.0)` will return `1n`, but `Nat(-1.0)` will throw). So I think this extra `insistVatID` is redundant, especially once we remove the prohibition on `v0`.